src/components/Footer.js

```diff
@@ -1,6 +1,6 @@
 import React from "react";
-const Footer = prop => {
-  const hotelInfo = prop.hotelContact;
+const Footer = ({ hotelContact }) => {
+  const hotelInfo = hotelContact;
   return (
     <ul>
       {hotelInfo.map((info, index) => (
```

Comment:
nice destructuring! you could make this more concise by using your hotelContact instead of hotelInfo in your map function (and then can get rid of your hotelInfo variable)